frontend/src/store/FamLoginUserState.ts

```diff
@@ -207,7 +207,61 @@ const getCachedAppRolesForDelegatedAdmin = (
             ? -1
             : 1;
     });
-}
+};
+
+const getMyCachedPermissions = () => {
+    let myRoles: any = [];
+    getUserAccess()?.map((item) => {
```

Comment:
I was checking the different between array.map vs array.forEach, I think if we're not returning any value, no need to use map?
<img width="724" alt="Screen Shot 2024-04-10 at 10 28 58 AM" src="https://github.com/bcgov/nr-forests-access-management/assets/23131735/f6446ded-a3c8-4445-81da-945fe640d931">

Reply:
hum...good to know, I'll use forEach in this case Thanks!